### DISCUSSION TITLE
k32 vs k33

### DISCUSSION THREAD

**Discussion by moshenskyDV** [Chia-Network/chia-blockchain#2929] [Q&A]
I've started this topic, because still unclear the best strategy of plots generating.

@hoffmang9 in his [article](https://www.chia.net/2021/02/22/plotting-basics.html) says that we have no profit of creating k33 plots.
According to [this topic](https://www.reddit.com/r/chia/comments/mqu3k8/k32_vs_k33/gvs2hmv?utm_source=share&utm_medium=web2x&context=3) on reddit user says that chance to win if you got a lottery ticket twice higher if you have k33.

So my assumptions:
1) k32 and k33 have the same chance to "get lottery ticket" (pass filter).
2) k33 has x2 change to win the lottery if k33 and k32 got lottery tickets.

So actually make sense to choose strategy of creating k33 plots? Could anyone of the officials confirm of decline this assumption?

By [chia explorer](https://www.chiaexplorer.com/charts/size) more than 90% users are choosing k32 size

**Comment by cccat6** [comment]
Pass the filter doesn't matter.
It already clear that the chance you win **ONLY** depends on the percentage you have in the total network space.
A k33 is about equivalent to 2*k32.

**Reply by cccat6** [reply]
By the way, @hoffmang9 is the official. He is the big guy in Chia company.

**Reply by moshenskyDV** [reply]
> Pass the filter doesn't matter.
> It already clear that the chance you win **ONLY** depends on the percentage you have in the total network space.
> A k33 is about equivalent to 2*k32.

Any proofs or official docs? I believe you, but I've heard a hundreds of opinions before I decided to create this topic, I want some proofs.

**Reply by cccat6** [reply]
https://www.chia.net/2021/02/22/plotting-basics.html
In the third paragraph. It claims that "You really never need to plot a plot with a k size larger than 32."

**Reply by moshenskyDV** [reply]
@cccat6 I mean more explanation (not proofs, it's too rude, sry) why the only space matter.

**Reply by cccat6** [reply]
Chia is actually designed in that way, which is only space matter. That is the root design. Everything else is depending on this setting.

**Reply by Last-Airbender** [reply]
There's no data yet on how many k32 or k33 plots winning ratio compared to the total netspace. The developers should open this kind of data or at least share the data with 3rd party like chia explorer. For the time being some people made k33 plots just trying to get a better result for this lottery.

**Reply by cccat6** [reply]
Oh my god, it is not a statistic. It is the design. If you have any concerns you can read the code. Chia is fully open source.
No doubt that there is no difference between different k values.
Why don't you guys just not believe the root design and try to use the statistical method to figure what it is?
The reason why it does not share the data because there is no necessary to do that. It is definitely meaningless.
K33 has twice of chances to win just because it two times larger than k32. Not for anything else like passing the filter. The filter is just for a quick check and do not need to waste resources on checking every plot. If you like, you can modify the code to make every block pass the filter but there is no difference.
You have to understand what is actually happening in the mining process. Not guessing at all. If you have any concern that what people in the forum is not true. READ THE CODE! Code shows everything.

**Reply by moshenskyDV** [reply]
@cccat6 I'm asking for explanation, because I'm as most of all regular miners not really qualified in python, most of all users even not qualified at programming at all.

**Reply by cccat6** [reply]
So I already explained. It is a **SETTING**.
What setting means is this is the rule. You have no reason to challenge the rule itself.
The rule is not a variable that needs to measure. It is the constant that will tell you directly.
If anyone not following Chia's rule. This person is not doing actual Chia.
Your question for me is just like "why should I eat and drink to live?" It is the setting of natural law.
And winning chance only depends on the space you have compared with total network space is the law in Chia world!

**Reply by chianew** [reply]
wa can find that "k=32 is now the minimum plot size that will be eligible for mainnet at the end of the year." on the FAQ- k sizes page.   Does it mean we have to delete K32 plots next year?

**Reply by hoffmang9** [reply]
No - "end of the year" in that context was the launch date of mainnet - which is now past. k32 should be fine for 7-15 years.
